bot/utils/time.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+
+from dateutil.relativedelta import relativedelta
+
+
+# All these functions are from https://github.com/python-discord/bot/blob/master/bot/utils/time.py
+def _stringify_time_unit(value: int, unit: str) -> str:
```

Comment:
Arrow library has `humanize()` function which can do this.

Reply:
See: https://github.com/python-discord/sir-lancebot/pull/462#discussion_r496356447